airbyte-webapp/src/packages/cloud/views/auth/components/PersonQuoteCover/PersonQuoteCover.module.scss

```diff
@@ -26,8 +26,8 @@
   background-image: linear-gradient(
     45deg,
     colors.$orange-400 10%,
-    rgba(colors.$orange-400, 0.8) 30%,
-    rgba(colors.$blue-400, 0.8) 80%,
+    rgba(255 106 77 / 80%) 30%,
+    rgba(97 94 255 / 80%) 80%,
```

Comment:
These two colors are kind of one-off colors, so I felt better defining them inline here than in a global variable.